.github/workflows/ci.yaml

```diff
@@ -26,6 +26,39 @@ jobs:
         command: lint
         config: .github/ct.yaml
 
+    - name: Check if jsonnet folder changed
+      uses: dorny/paths-filter@v2
+      id: changes
+      with:
+        filters: |
+          jsonnet:
+            - 'jsonnet/**'
```

Comment:
Remove indent before '-' to be consistent with rest of the code.

Reply:
done